libsql-server/src/stats.rs

```diff
@@ -22,6 +43,11 @@ pub struct Stats {
     write_requests_delegated: AtomicU64,
     #[serde(default)]
     current_frame_no: AtomicU64,
+    // Lowest value in currently stored top queries
+    #[serde(default)]
+    top_query_threshold: AtomicI64,
```

Comment:
We don't need that, but I want to avoid taking a lock for `top_queries` to peek `first()`, if a query doesn't qualify in the first place. It's just fast path optimization, assuming that 99.999% of the queries are short and don't need to be recorded in the `top_queries` ranking.